nanome/_internal/_network/_commands/_serialization/_upload_cryo_em.py

```diff
@@ -0,0 +1,20 @@
+from nanome._internal._network._serialization import _ContextDeserialization, _ContextSerialization
+from nanome._internal._volumetric._serialization import _VolumeDataSerializer
+from nanome._internal._util._serializers import _TypeSerializer
+from nanome._internal._volumetric._io._em_map._parse import parse_file
+class _UploadCryoEM(_TypeSerializer):
+    def __init__(self):
+        self.vd_serializer = _VolumeDataSerializer()
+
+    def version(self):
+        return 0
+
+    def name(self):
+        return "UploadCryoEM"
+
+    def serialize(self, version, value, context):
+        volume_data = parse_file(value)
```

Comment:
You should probably check for volume_data == None, and, if true, display and error message and return properly? Or maybe just send an empty map.
But I think, for now, the _VolumeDataSerializer serializer is going to crash